en/docs/concepts/authentication/back-channel-logout.md

```diff
@@ -0,0 +1,22 @@
+# OpenID Connect Back-Channel Logout
+
+[OpenID Connect back-channel logout](https://openid.net/specs/openid-connect-backchannel-1_0.html#Backchannel) feature
+enables logging out users from a client application/Relying Party (RP) by directly communicating the logout requests 
+between the client application and authorization server.
+
+## What is direct communication?
+
+Direct communication enables communicating the requests between the client application and authorization server through 
+direct network links without having to rely on a browser/agent. This approach is more reliable as it does not require
+communicating the logout requests through a user agent/browser and maintain active RP browser sessions for the 
+communication to succeed.
+
+## Message flow¶
+Let's take a look at the underlying message flow of the OpenID Connect back-channel logout.
```

Comment:
The "the" in the phrase "the OpenID Connect back-channel logout" is not required